adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -2749,7 +2749,7 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
   /**
    * Load variants into a VariantRDD.
    *
-   * If the path name has a .vcf/.vcf.gz/.vcf.bgzf/.vcf.bgz extension, load as VCF format.
```

Comment:
Due to Hadoop codec limitations, we don't support `.bgzf` on load. We support it on write since we choose the output codec, but there's no `.bgzf` codec and the codecs are autodetected on read from the file extensions.